### PR TITLE
MODORDSTOR- 83- acquisition unit membership schema

### PR DESCRIPTION
**Pull request by KVupp**
## Purpose
https://issues.folio.org/browse/MODORDSTOR-83

## Approach

- add new schemas to support membership records

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Examples exist for all schemas
  - [x] Descriptions exist for all schema properties
  - [x] All schemas pass raml-cop linting
- Does this introduce breaking changes?
  - [ ] Were there any schema changes?
  - [x] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: mod-orders-storage/schemas/acquisitions_unit_membership.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Relationship between a user and acquistionUnit",
```
Comment (grabsefx) [inline mod-orders-storage/schemas/acquisitions_unit_membership.json]: nitpick. I think `acquistionUnit` should be splitted into separate words

File: mod-orders-storage/examples/acquisitions_unit_membership_collection.sample [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+     {
+        "id": "567a6d67-460f-43ab-af28-8c730c9aa4da",
+        "userId": "480dba68-ee84-4b9c-a374-7e824fc49227",
+        "acquisitionsUnitId": "f6d2cc9d-82ca-437c-a4e6-e5c30323df00"
+     },
+     {
+        "id": "874225ae-e3f5-4666-ae19-b35709ff0ee9",
+        "userId": "301883b4-a87b-4fc8-a4c1-e13813db06b0",
+        "acquisitionsUnitId": "aa0ec4e1-782f-45f6-a6f3-8e6b6c00599c"
+     },
```
Comment (RamanAsadchy) [inline mod-orders-storage/examples/acquisitions_unit_membership_collection.sample]: Could you remove the last comma, please.

File: mod-orders-storage/schemas/acquisitions_unit_membership.json (hunk 1 above)
Comment (KVupp) [reply mod-orders-storage/schemas/acquisitions_unit_membership.json]: done

File: mod-orders-storage/examples/acquisitions_unit_membership_collection.sample (hunk 2 above)
Comment (KVupp) [reply mod-orders-storage/examples/acquisitions_unit_membership_collection.sample]: done

**Pull request merged**